### PR TITLE
Ability to change reward

### PR DESCRIPTION
**Pull request by obaranni**

### CODE REVIEW

File: sources/scripts.move [hunk 1]
```diff
@@ -149,6 +158,21 @@ module harvest::scripts {
...
+    ///     * `users` - list of users to whitelist.
+    public entry fun add_into_whitelist<S, R>(pool_owner: &signer, users: vector<address>) {
+        stake::add_into_whitelist<S, R>(pool_owner, users);
+    }
+
+    /// Remove user from whitelist.
+    ///     * `owner` - pool creator account.
+    ///     * `user` - address of user to remove from whitelist.
+    /// Note: If no users left in whitelist it become deactivated.
+    public entry fun remove_from_whitelist<S, R>(pool_owner: &signer, user: address) {
```
Comment (borispovod) [inline sources/scripts.move:172]: Can we have also `vector` here?

File: sources/stake.move [hunk 2]
```diff
@@ -225,62 +243,128 @@ module harvest::stake {
...
     ///     * `depositor` - rewards depositor account.
     ///     * `pool_addr` - address under which pool are stored.
     ///     * `coins` - R coins which are used in distribution as reward.
-    public fun deposit_reward_coins<S, R>(depositor: &signer, pool_addr: address, coins: Coin<R>) acquires StakePool {
+    ///     * `duration` - new pool life duration.
+    public fun deposit_reward_coins<S, R>(
+        depositor: &signer,
+        pool_addr: address,
+        coins: Coin<R>,
+        duration: u64,
```
Comment (borispovod) [inline sources/stake.move:297]: Can we have also start of epoch?

**Review by borispovod — CHANGES_REQUESTED** [review]

File: sources/scripts.move (hunk 1 above)
Comment (obaranni) [reply sources/scripts.move:172]: I don't see the need, but we can

File: sources/stake.move (hunk 2 above)
Comment (obaranni) [reply sources/stake.move:297]: Sounds like a big deal, I can't say for sure

Comment (borispovod) [reply sources/stake.move:297]: if it's too much changes it's ok

File: sources/scripts.move (hunk 1 above)
Comment (borispovod) [reply sources/scripts.move:172]: Ok

**Pull request merged**